src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/model/EmailHistory.java

```diff
@@ -21,50 +26,79 @@
 @Setter
 @NoArgsConstructor
 public class EmailHistory extends Auditable<String> {
-	
+
 	/**
 	 * Primary key
 	 */
 	@Schema(description = "ID", accessMode = Schema.AccessMode.READ_ONLY)
 	@Id
 	@GeneratedValue
 	private Long emailHistoryId;
-	
+
 	/**
-	 * FromEmailAddress.
+	 * The date and time the email was sent
 	 */
-	@Column(length = 500)
+	@Column
+	@Temporal(TemporalType.TIMESTAMP)
 	@Schema(nullable = false)
+	private Date emailSentTs;
+
+	/**
+	 * FromEmailAddress.
+	 */
+	@Column(length = 100)
+	@Size(max = 100)
+	@Schema(maxLength = 100 ,nullable = false)
 	private String fromEmailAddress;
-	
+
 	/**
 	 * ToEmailAddress.
 	 */
-	@Column(length = 500)
-	@Schema(nullable = false)
+	@Column(length = 4000)
+	@Size(max = 4000)
+	@Schema(maxLength = 4000, nullable = false)
```

Comment:
4000 characters? Please confirm. The OCCAM_DISPUTES.EMAIL_ADDRESS_TXT field is only 100 characters. I would think this field is copy of that field and should match.

Reply:
Yeah, I was surprised about that too but I have already confirmed that the database field for `toEmailAddress` in Oracle actually holds 4000 characters. I thought the same thing that it may hold a collection of email addresses, very likely that it could be the reason. That might be a standard design for Justin Oracle database so I will leave that as it is.

I think our application supports a sending email to multiple email addresses but I can't think of any use case for that right now.